src/main/java/com/arpnetworking/tsdcore/sinks/DimensionFilteringSink.java

```diff
@@ -33,6 +34,12 @@
      */
     @Override
     public void recordAggregateData(final PeriodicData data) {
+        if (_whitelistDimensions.isPresent()) {
+            if (!_whitelistDimensions.get().containsAll(data.getDimensions().keySet())) {
```

Comment:
Indeed, I would expect that metrics from older MAD installations would have no dimensions at all, and the service, host and cluster would be in the FQDSN or similar...?